monai/bundle/utils.py

```diff
@@ -233,3 +242,22 @@ def load_bundle_config(bundle_path: str, *config_names: str, **load_kw_args: Any
             parser.read_config(f=cdata)
 
     return parser
+
+
+def merge_kv(args: dict | Any, k: str, v: Any) -> None:
+    """
+    Update the `args` dict-like object with the key/value pair `k` and `v`.
+    """
+    if k.startswith(MERGE_KEY):
+        id = k[1:]
+        if id in args and v is not None:
+            if isinstance(v, dict) and isinstance(args[id], dict):
+                args[id].update(v)
+            elif isinstance(v, list) and isinstance(args[id], list):
```

Comment:
I would love we add some doc string or doc somewhere to illustrate two things before users meet the error:
1. The `+` symbol only support dict and list.
2. The `+` symbol will merge keys in the dict and append the list.

Thanks,  
Bin

Reply:
Fixed.